portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/PublicationsTemplate.testcase

```diff
@@ -216,6 +216,41 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LPS-171576 - The publication template selection is gone for a created publication"
+	@priority = 3
+	test CanSelectAnotherPublicationTemplateAfterEditingPublication {
+		task ("Create Publications Templates and one Publication with template") {
+			PublicationsTemplate.gotoPublicationTemplate();
+
+			for (var order : list "1,2") {
+				PublicationsTemplate.addPublicationTemplate(
+					publicationName = "P${order}",
+					templateName = "Publication Template ${order}");
+			}
+
+			Publications.createNewPublicationWithTemplate(
+				gotoPublicationsAdmin = "true",
+				publicationName = "P1",
+				templateName = "Publication Template 1");
+		}
+
+		task ("Go to edit Publication and select another template") {
```

Comment:
Usually, we should add a step to navigate to Publications admin page to avoid the flaky issue when editing the publication, can you add the step to navigate Publications page `PublicationsNavigator.openPublicationsAdmin();` before editing it?